sysutils/rename/Portfile

```
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        ap rename 0da9faa241093c90a40b15a9ff4b36e9858d4516
+name                rename
+version             1.601
+revision            0
+categories          sysutils
+license             GPL-1+
+maintainers         @dylanarmstrong
```

Ports should also be marked `openmaintainer` unless there is a very specific reason why you, and only you should be the only one to maintain this port:

```suggestion
maintainers         @dylanarmstrong openmaintainer
```